GHzDACs/jumpTable.py

```diff
@@ -255,28 +360,58 @@ def testNormal(stopAddr):
     of this.1
     
     We observe in this case that if we specify 1 repetition, the system instead
-    executes many times. Why?
+    executes many times. Why? -- this is no longer the case??
     
     If stopAddr==3
     ???
     """
     waveform = np.zeros(256)
-    waveform[4:12] = 0.8
-    waveform[20:24] = 0.8
-    
+    waveform[40:120] = 0.2
+    waveform[200:220] = 0.4
+    # waveform[44:48] = 0.4
+    # waveform[48:60] = 0.3
+
     jumpEntries = []
-    
-    #End execution
+
+    # End execution
     op = END()
     fromAddr = stopAddr
-    toAddr = 0 #Meaningless?
+    toAddr = 0              # Should be 0 for END and other ops w/o toAddrs
     jumpEntries.append(JumpEntry(fromAddr, toAddr, op))
-    
+
     table = JumpTable(0)
     table.jumps = jumpEntries
```

Comment:
I would prefer to make these classes immutable and just use the constructor to create them:
```python
table = JumpTable(start_addr=0, jumps=jumpEntries)
```